lib/ansible/modules/cloud/google/gcp_compute_address.py

```diff
@@ -32,85 +32,92 @@
-              except the last character, which cannot be a dash.
-        required: true
-    network_tier:
-        description:
-            - 'The networking tier used for configuring this address. This field can take the
-              following values: PREMIUM or STANDARD. If this field is not specified, it is assumed
-              to be PREMIUM.'
-        required: false
-        version_added: 2.8
-        choices: ['PREMIUM', 'STANDARD']
-    subnetwork:
-        description:
-            - The URL of the subnetwork in which to reserve the address. If an IP address is specified,
-              it must be within the subnetwork's IP range.
-            - This field can only be used with INTERNAL type with GCE_ENDPOINT/DNS_RESOLVER purposes.
-            - 'This field represents a link to a Subnetwork resource in GCP. It can be specified
-              in two ways. You can add `register: name-of-resource` to a gcp_compute_subnetwork
-              task and then set this subnetwork field to "{{ name-of-resource }}" Alternatively,
-              you can set this subnetwork to a dictionary with the selfLink key where the value
-              is the selfLink of your Subnetwork.'
-        required: false
-        version_added: 2.7
-    region:
-        description:
-            - URL of the region where the regional address resides.
-            - This field is not applicable to global addresses.
-        required: true
+    - Whether the given object should exist in GCP
+    choices:
+    - present
+    - absent
+    default: present
+  address:
+    description:
+    - The static external IP address represented by this resource. Only IPv4 is supported.
+      An address may only be specified for INTERNAL address types. The IP address
+      must be inside the specified subnetwork, if any.
+    required: false
+  address_type:
+    description:
+    - The type of address to reserve, either INTERNAL or EXTERNAL.
+    - If unspecified, defaults to EXTERNAL.
+    required: false
+    default: EXTERNAL
+    version_added: 2.7
+    choices:
+    - INTERNAL
+    - EXTERNAL
+  description:
+    description:
+    - An optional description of this resource.
+    required: false
+  name:
+    description:
+    - Name of the resource. The name must be 1-63 characters long, and comply with
+      RFC1035. Specifically, the name must be 1-63 characters long and match the regular
+      expression `[a-z]([-a-z0-9]*[a-z0-9])?` which means the first character must
+      be a lowercase letter, and all following characters must be a dash, lowercase
+      letter, or digit, except the last character, which cannot be a dash.
+    required: true
+  network_tier:
+    description:
+    - 'The networking tier used for configuring this address. This field can take
+      the following values: PREMIUM or STANDARD. If this field is not specified, it
+      is assumed to be PREMIUM.'
+    required: false
+    version_added: 2.8
```

Comment:
Looks like you've added new features here, so this is a `feature` PR, so can't backport it